doc/source/whatsnew/v1.1.0.rst

```diff
@@ -755,7 +755,7 @@ ExtensionArray
 - Fixed bug where :meth:`Series.value_counts` would raise on empty input of ``Int64`` dtype (:issue:`33317`)
 - Fixed bug in :class:`Series` construction with EA dtype and index but no data or scalar data fails (:issue:`26469`)
 - Fixed bug that caused :meth:`Series.__repr__()` to crash for extension types whose elements are multidimensional arrays (:issue:`33770`).
-
+- Fixed bug where :meth:`init_dict` would raise on empty input (:issue:`27953` and :issue:`33623`)
```

Comment:
always make this user facing. init_dict is a private method. likely you want ```DataFrame(columns=.., dtype='string')``` would fail

Reply:
Thanks @jreback .  I updated. Please review.